Docs/Configuration.tex

```diff
@@ -575,31 +572,31 @@ \subsection{Installation and Upgrade}\label{configuration-install}
 the \hyperref[miscsecurityprops]{Security Properties} section of this document.
 
 The \texttt{OC\ config} file, as with any property list file, can be edited with
-any text editor such as nano and vim. However, specialised software may provide
-a better experience. On macOS, the preferred GUI application is
+any text editor such as TextEdit and NotePad. However, specialised software
```

Comment:
(Regardless of editor discussion!) I would throw in a comma and "or" instead of "and" - but again a tiny, probably preference thing.

```
The \texttt{OC\ config} file, as with any property list file, can be edited with
any text editor, such as TextEdit or NotePad. However, specialised software
```